torch/_dynamo/variables/torch_function.py

```diff
@@ -189,9 +199,26 @@ def get_mode_index(cls, ind):
         return ind + cls.offset
 
 
-class TorchFunctionModeVariable(ContextWrappingVariable):
+class TorchFunctionModeVariable(GenericContextWrappingVariable):
```

Comment:
Out of curiosity - what cases require us to use `GenericContextWrappingVariable` over `ContextWrappingVariable`?

Reply:
We need the UDO behavior, sometimes there are custom methods on the mode that get called. I feel like this is happening in other places in dynamo too (ie subclassing dict) we might need to start thinking about a better way to have UDO like behavior on other types, but went with this.